vue-components/src/components/InputWithExtender.vue

```diff
@@ -109,6 +109,14 @@ export default defineComponent( {
 		border-radius: $wikit-InputExtender-border-radius;
 	}
 
+	&__placeholder {
```

Comment:
What is the motivation behind this second class instead of adding those styles to `wikit-InputWithExtender__extension`?